docs/src/app/logo.tsx

```diff
@@ -0,0 +1,90 @@
+import { cn } from 'fumadocs-ui/utils/cn';
+
+export function LogoWithSolana({
```

Comment:
Is this (and maybe others) also something that could just straight up be implemented with `next/image` through:

```ts
import { Image } from 'fumadocs-core/framework';
```

Reply:
Here again it means we have to hardcode colour values instead of relying on CSS variable via Tailwind. Is `next/image` providing any performance benefits on SVGs?